app.js

```diff
@@ -0,0 +1,33 @@
+require("dotenv").config();
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+
+const { AppDataSource } = require("./src/models/data-source")
+const { routes } = require("./src/routes");
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("combined"));
+
+app.use(routes);
+
+
+app.get('/ping'  , function (req,res ,next){
+  res.json({message :'pong'})
+})
+
+const startServer = async () => {
+  const PORT = process.env.PORT;
+
+  await AppDataSource.initialize();
```

Comment:
```suggestion
  await appDataSource.initialize();
```
appDataSource.initialize() 함수가 실행되고 나면, then()과 catch()를 사용해서 에러를 핸들링해줄 수 있습니다. 이렇게 하면, initialize() 함수가 실행되어서 database와 실제로 connection을 생성할 때, 성공했는지 실패했는지 여부를 확인할 수 있습니다. 

위에 해당하는 .then().catch()코드가 추가되어야겠습니다.